package.json

```diff
@@ -26,43 +26,44 @@
     "url": "https://github.com/rangle/angular-2-aot-sandbox/issues"
   },
   "dependencies": {
```

Comment:
These are the most up to date deps I could use that didn't cause peerDependency warnings... the latest version of `@ngtools/webpack` has a peerDependency on v2.2.1 of the angular libs.